src/locales/en/registerForm.json

```diff
@@ -0,0 +1,15 @@
+{
+  "registerTitles": ["Create your account", "REGISTER"],
```

Comment:
let's create a `title` and `shortTitle` instead of array of tittles

Reply:
should be fixed in 9e244cb